go/vt/vtgate/executor.go

```diff
@@ -280,6 +284,22 @@ func (e *Executor) handleSet(ctx context.Context, session *vtgatepb.Session, sql
 			default:
 				return nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "unexpected value for client_found_rows: %d", val)
 			}
+		case "no_cache_query_plan":
```

Comment:
Although this was the option we discussed IRL, as a general rule I try to avoid having the "negative" case encoded in options, even when it is the default.

In other words, I think we should change this so the exposed option is `set cache_query_plan` with a default value of 1.